--- ltconfig.orig	Tue Jul  6 15:55:36 1999
+++ ltconfig	Tue Jul  6 15:59:12 1999
@@ -1200,8 +1200,8 @@ EOF
   esac
 
   if test "$ld_shlibs" = yes; then
-    runpath_var=LD_RUN_PATH
-    hardcode_libdir_flag_spec='${wl}--rpath ${wl}$libdir'
+    runpath_var=LD_LIBRARY_PATH
+    hardcode_libdir_flag_spec=''
     export_dynamic_flag_spec='${wl}--export-dynamic'
     case $host_os in
     cygwin* | mingw*)
